client/app/scripts/superdesk-archive/related-item-widget/relatedItem.js

```diff
@@ -0,0 +1,73 @@
+define([
+    'angular',
+    'require',
+    'moment',
+    'superdesk-archive/archive-widget/baseWidget'
+], function(angular, require, moment) {
+    'use strict';
+
+    angular.module('superdesk.widgets.relatedItem', [
+        'superdesk.widgets.base',
+        'superdesk.authoring.widgets'
+    ])
+    .config(['authoringWidgetsProvider', function(authoringWidgets) {
+            authoringWidgets.widget('related-item', {
+                label: gettext('Related Item'),
+                icon: 'multiedit',
+                template: require.toUrl('./widget-relatedItem.html'),
+                side: 'right',
+                display: {authoring: true, packages: false}
+            });
+        }])
+        .controller('relatedItemController', ['$scope', 'api', 'BaseWidgetController', '$location',
+        function ($scope, api, BaseWidgetController, $location) {
+            var todayDateTime = new Date();
+            var before24HrDateTime = new Date(todayDateTime.setDate(todayDateTime.getDate() - 1)).toISOString();
```

Comment:
I think there is some unnecessary juggling here.
Maybe this would be sufficient:
```
var before24HrDateTime = new Date(new Date() - 1).toISOString();
```
(or something similar)